From dbbb19b8e412fb815d9eadceb1c0ce40083a5504 Mon Sep 17 00:00:00 2001
From: Tsz Shan Chan <tsz.chan.dev@gmail.com>
Date: Fri, 31 Jul 2026 17:18:45 +1000
Subject: [PATCH 10490/15333] pinctrl: sx150x: allow build when I2C is a module

PINCTRL_SX150X currently depends on I2C=y. This prevents the driver
from being built when I2C is configured as module.

Change the Kconfig dependency to just I2C so sx150x can be built as a
module when I2C is also a module.

Signed-off-by: Tsz Shan Chan <tchan@jacques.com.au>
Signed-off-by: Linus Walleij <linusw@kernel.org>
---
 drivers/pinctrl/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/pinctrl/Kconfig b/drivers/pinctrl/Kconfig
index ddc60562b770..fdcd0ea34d6f 100644
--- a/drivers/pinctrl/Kconfig
+++ b/drivers/pinctrl/Kconfig
@@ -575,7 +575,7 @@ config PINCTRL_STMFX
 
 config PINCTRL_SX150X
 	tristate "Semtech SX150x I2C GPIO expander pinctrl driver"
-	depends on I2C=y
+	depends on I2C
 	select PINMUX
 	select PINCONF
 	select GENERIC_PINCONF
-- 
2.55.0

